### PR TITLE
Add Jmeter / ansible performance tests to jenkins

### PR DESCRIPTION
**Pull request by cthayes**
This PR has most of the work to integrate the technologies necessary to run the performance tests from jenkins. Some of the changes to jmeter may not be necessary, since I  was trying several things to get it to work properly. Some cleanup is needed. There are also further tweaks to be made as part of other tickets. 

BLUEBUTTON-758
BLUEBUTTON-220

### CODE REVIEW

**Review by dho937 — APPROVED** [review]
Wow...impressive....

File: ansible/vars/dpr.yml
Comment (jcgeer) [inline ansible/vars/dpr.yml:4]: Does this mean it is specifically tied to your MFA device/profile? In the future will we have to change this at all? If so, is that documented anywhere?

**Review by jcgeer — APPROVED** [review]
I think this looks good (no obvious errors) though I'm don't entirely understand it all. I did post a comment just to get some clarification on something, but otherwise good to go.

File: ansible/vars/dpr.yml
Comment (cthayes) [reply ansible/vars/dpr.yml:4]: I actually don't think that's doing anything. I'll clean it up in the next PR waiting for this to merge.

**Comment by cthayes** [comment]
> I think this looks good (no obvious errors) though I'm don't entirely understand it all. I did post a comment just to get some clarification on something, but otherwise good to go.

Once the next couple PRs are merged and the tests are stabilized I can go through the technical pieces with the team.

**Pull request merged**
